apps/api/src/controllers/stateManager.ts

```diff
@@ -241,11 +241,21 @@ export class ApplicationStateManager extends StateMachine<ApplicationStateValues
 	}
 
 	async closeRepReview() {
-		return this.closeDraft();
+		const transitionResult = this._canPerformAction(close);
+		if (transitionResult.success) {
+			return await this._dispatchAndUpdateAction(close, 'close');
+		} else {
+			return failure(`Cannot close application with state ${this.getState()}`);
+		}
 	}
 
 	async closeDacReview() {
-		return this.closeDraft();
+		const transitionResult = this._canPerformAction(close);
+		if (transitionResult.success) {
+			return await this._dispatchAndUpdateAction(close, 'close');
+		} else {
+			return failure(`Cannot close application with state ${this.getState()}`);
+		}
```

Comment:
The `close` functions all have the same functionality at the moment, but if they all call `closeDraft` then there's no separation btwn one `close` method and another

This change enables us to update the individual `close` functions as needed but they still have the core `close` action